docs/contribute/core.md

```diff
@@ -64,6 +64,33 @@ The following is the `.gitignore` file in the `asdf-vm/asdf` repository. We igno
 
 @[code](../../.gitignore)
 
+### `.git-blame-ignore-revs`
+
+`asdf` uses a `.git-blame-ignore-revs` to reduce noise when running a blame.
```

Comment:
After this line, can you link to the official Git documentation for this feature?